build.gradle.kts

```diff
@@ -30,7 +30,7 @@ sonarqube {
         property("sonar.organization", "navit")
         property("sonar.host.url", "https://sonarcloud.io")
         property("sonar.login", System.getenv("SONAR_TOKEN"))
-        property("sonar.exclusions", "**/Koin*,**Mock**")
+        property("sonar.exclusions", "**/Koin*,**Mock**,**/App**")
```

Comment:
Anbefalt måte å få høyere testdekning :-D

Reply:
Ja dette er en synd/test for å se om den funker, har prøvd å ha den før men da kicka den ikke inn.

Vi får ikke testet App.kt 100% før vi kjører opp en e2e testsuite, og det blir down the road :)